gloo/common/utils.cc

```diff
@@ -11,6 +11,7 @@
 #ifdef _WIN32
 #include <winsock2.h>
 #else
+#include <stdlib.h>
```

Comment:
On Linux with `clang`, I find that this only works if I use `<cstdlib>` here.

```suggestion
#include <cstdlib>
```

If that also works in your environment, I would recommend changing it?

Reply:
This should be the case, let's use it